### PR TITLE
Web fonts - font is downloaded if any characters are in unicode range

### LINKED ISSUE #29403: Ambiguous text
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts

### What specific section or headline is this issue about?

the description of unicode-range in the last **Note** block.

### What information was incorrect, unhelpful, or incomplete?

the referance page of the unicode-range CSS descriptor https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range says "If the page doesn't use any character in this range, the font is not downloaded; if it uses at least one, the whole font is downloaded.",but the guide said "in supporting browsers, only the specified characters will be downloaded".I don't know which is correct.

### What did you expect to see?

i want you to correct it.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/css/styling_text/web_fonts`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/styling_text/web_fonts/index.md
* Last commit: https://github.com/mdn/content/commit/45268b07c84a04b45d46bcdf104e2b33be00adcf
* Document last modified: 2023-07-17T22:36:05.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #29403

This previously stated that only specified chars will be downloaded from unicode range will be downloaded, but as per spec and https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range the range indicates the chars that must appear in the page for the font to be downloaded

### CODE REVIEW

File: files/en-us/learn/css/styling_text/web_fonts/index.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ Let's go through it to see what it does:
...
+> **Note:** You can also specify particular {{cssxref("font-variant")}} and {{cssxref("font-stretch")}} values for your web fonts. In newer browsers, you can also specify a {{cssxref("@font-face/unicode-range", "unicode-range")}} value, which is a specific range of characters you want to use out of the web font — in supporting browsers, only fonts that contain those specified characters will be downloaded, saving unnecessary downloading. [Creating Custom Font Stacks with Unicode-Range](https://24ways.org/2011/creating-custom-font-stacks-with-unicode-range/) by Drew McLellan provides some useful ideas on how to make use of this.
```
Comment (hamishwillee) [inline files/en-us/learn/css/styling_text/web_fonts/index.md:196]:
```suggestion
> **Note:** You can also specify particular {{cssxref("font-variant")}} and {{cssxref("font-stretch")}} values for your web fonts. In newer browsers, you can also specify a {{cssxref("@font-face/unicode-range", "unicode-range")}} value, which is a specific range of characters you might want to use out of the web font — in supporting browsers, the font will only be downloaded if the page contains those specified characters, saving unnecessary downloading. [Creating Custom Font Stacks with Unicode-Range](https://24ways.org/2011/creating-custom-font-stacks-with-unicode-range/) by Drew McLellan provides some useful ideas on how to make use of this.
```

File: files/en-us/learn/css/styling_text/web_fonts/index.md [hunk 2]
```diff
@@ -193,7 +193,7 @@ Let's go through it to see what it does:
...
+> **Note:** You can also specify particular {{cssxref("font-variant")}} and {{cssxref("font-stretch")}} values for your web fonts. In newer browsers, you can also specify a {{cssxref("@font-face/unicode-range", "unicode-range")}} value, which is a specific range of characters you might want to use out of the web font — in supporting browsers, the font will only be downloaded if the page contains those specified characters, saving unnecessary downloading. [Creating Custom Font Stacks with Unicode-Range](https://24ways.org/2011/creating-custom-font-stacks-with-unicode-range/) by Drew McLellan provides some useful ideas on how to make use of this.
```
Comment (chrisdavidmills) [inline files/en-us/learn/css/styling_text/web_fonts/index.md:196]:
```suggestion
> **Note:** You can also specify particular {{cssxref("font-variant")}} and {{cssxref("font-stretch")}} values for your web fonts. In newer browsers, you can also specify a {{cssxref("@font-face/unicode-range", "unicode-range")}} value, which is a specific range of characters you might want to use out of the web font. In supporting browsers, the font will only be downloaded if the page contains those specified characters, saving unnecessary downloading. [Creating Custom Font Stacks with Unicode-Range](https://24ways.org/2011/creating-custom-font-stacks-with-unicode-range/) by Drew McLellan provides some useful ideas on how to make use of this.
```

Small change, just to cut down on sentence length.

**Review by chrisdavidmills — APPROVED** [review]
LGTM; thanks @hamishwillee!

Approving, but I did have a small suggestion.

File: files/en-us/learn/css/styling_text/web_fonts/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/learn/css/styling_text/web_fonts/index.md:196]: I thought about this and decided to keep the emdash, but mostly to reduce churn. I prefer this :-)

**Pull request merged**